selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml

```diff
@@ -165,6 +171,7 @@
             <class name="org.eclipse.che.selenium.factory.CheckFactoryWithUntilPolicyTest"/>
             <class name="org.eclipse.che.selenium.factory.CheckOpenFileFeatureTest"/>
             <class name="org.eclipse.che.selenium.factory.CheckRunCommandFeatureTest"/>
+            <class name="org.eclipse.che.selenium.git.CommitFilesByMultiSelectTest"/>
```

Comment:
I guess this is won't help if I try to run all test from the org.eclipse.che.selenium.git package

Reply:
Yes, but in that case user should run them with `--thread=1`.

Reply:
Any way, why do git tests fail if they run simultaneously?

Reply:
Because of setup methods operate with ssh-keys and application grants for github.com

Reply:
So, since method is synchronized no issues occur then?

Reply:
Probably yes, but it's better to avoid parallel execution of these tests anyway so as github.com and git-related properties are the same for all test workspaces and potentially git-tests can influence each other when they run in parallel.

Reply:
These changes confuse a lot. Shouldn't be done.

Reply:
That is the limitation of having all tests in one **\<test>** section of _suite-xml_. We do need start git-related tests in one thread. Alternatively,  we can set change _parallel_-mode of test execution to run all tests within the one **\<test>** section one-by-one, but it requires extracting all other tests into the separate **\<test>** sections as well, and it also will increase the time of all tests execution.

Reply:
We can try to extract tests of git into separate suite file as described [here](https://stackoverflow.com/a/12300270), but it looks like undocumented feature so as such ability isn't mentioned on [official site of testng](http://testng.org/doc/documentation-main.html).